l10n_it_withholding_tax/views/account.xml

```diff
@@ -213,20 +132,17 @@
         -->
         <record model="ir.ui.view" id="view_wt_invoice_line_form">
             <field name="name">view.withholding.invoice.supplier.form</field>
-            <field name="model">account.invoice.line</field>
-            <field name="inherit_id" ref="account.view_invoice_line_form" />
+            <field name="model">account.move.line</field>
+            <field name="inherit_id" ref="account.view_move_line_form" />
             <field name="arch" type="xml">
 
-                <xpath
-                expr="//group/group/field[@name='invoice_line_tax_ids']"
-                position="after"
-            >
+                <xpath expr="//group/group/field[@name='tax_ids']" position="after">
                     <field
-                    name="invoice_line_tax_wt_ids"
-                    widget="many2many_tags"
-                    options="{'no_create': True}"
-                    invisible="context.get('type', 'out_invoice') not in ['in_invoice', 'in_refund']"
-                />
+                        name="invoice_line_tax_wt_ids"
+                        widget="many2many_tags"
+                        options="{'no_create': True}"
+                    />
+                    <!-- attrs="{'invisible': context.get('move_type', 'out_invoice') not in ['in_invoice', 'in_refund']}" -->
```

Comment:
come mai è stato commentato?

Reply:
Ripristinato come su 12.0 (pare che context.get() is un'eccezione alla regola) vd. Pedro

Reply:
Vedi https://odoo-community.org/groups/contributors-15/contributors-196808?mode=date&date_begin=2021-05-01&date_end=2021-06-01